### PR TITLE
Michael Ducey

### PR DESCRIPTION
**Pull request by mikeducey**

### CODE REVIEW

File: lib/3_add_up_numbers.js [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 module.exports = {
   add_up_numbers: function(arr){
-    // Your code goes here.
+    var total = arr.reduce(function(x, y) {
```
Comment (halfghaninne) [inline lib/3_add_up_numbers.js:9]: Please be mindful of indentation! _Within_ (ie indented one space more than) the outermost function, which begins on line 8, I'd expect to see indentation like so:

``` javascript
var total = arr.reduce(function(x, y) {
  return x + y;
});
return_total;
```

File: lib/4_filter_numbers.js [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+  function lessthanten(number) {
```
Comment (halfghaninne) [inline lib/4_filter_numbers.js:9]: Indentation issues in this solution, as well!

File: lib/2_uppercases_all_letters.js [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 module.exports = {
   uppercases_all_letters: function(str){
-    // Your code goes here.
+    string_upp = str.toUpperCase();
+      return string_upp
```
Comment (halfghaninne) [inline lib/2_uppercases_all_letters.js:10]: Just to be consistent 😄  Line 10 can be brought back one indentation to be in line with Line 9.

File: lib/4_filter_numbers.js (hunk 2 above)
Comment (halfghaninne) [inline lib/4_filter_numbers.js:9]: I like your instinct to split these up into two functions. Can you say more about why you chose to do this?

**Comment by halfghaninne** [comment]
👍  Nice work!

**Comment by mikeducey** [comment]
Hi Alexandra-  Sorry, I should have checked this much earlier.  I'll get
these indentation issues fixed (getting my last few things in order for the
program...)

**Pull request closed without merging**